test/resources/controllers/AlertControllerTest.testCreateInvalidPeriodCase.json

```diff
@@ -0,0 +1,21 @@
+{
+    "id" : "1a3d037c-ee11-11e1-a1e5-00259060b5c0",
```

Comment:
Looks like you have the same id in here for several creates. Is that intentional? Does it matter?